lnwallet/btcwallet/btcwallet.go

```diff
@@ -449,6 +450,18 @@ func (b *BtcWallet) PublishTransaction(tx *wire.MsgTx) error {
 		}
 		return err
 	}
+
+	switch b.chain.(type) {
```

Comment:
A better place to do this, is within the `PublishTransaction` method within `btcwallet` itself. That way, this is also fixed for all other users of the wallet, not only `lnd`.

Reply:
@Roasbeef I've created pull request there - https://github.com/btcsuite/btcwallet/pull/514 , this one I guess could be closed